features/step_definitions/means_report_steps.rb

```diff
@@ -1,3 +1,4 @@
+# TODO: can be deleted post-MTRA
```

Comment:
Add ticket(s) to the backlog, so these TODOs are not forgotten.

Reply:
will go through and add this ticket https://dsdmoj.atlassian.net/browse/AP-5493

Reply:
ticket number added to all relevant todos